src/odemis/gui/util/test/updater_test.py

```diff
@@ -24,10 +24,13 @@
 
 # test the functions of the gui.util.updater module
 from __future__ import division
-from odemis.gui.util import updater
+
 import logging
+from odemis.gui.util import updater
 import unittest
 
+import wx
+
```

Comment:
I don't know if that is a PEP8 rule but if I reorder the import I like to group and separate the odemis, external and future imports.
That isn't what happened with this PR though. Personally I would prefer something like::
```
from __future__ import division

import unittest
import wx
import logging

from odemis.gui.util import updater
```

Reply:
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
>     1. Standard library imports.
>     2. Related third party imports.
>     3. Local application/library specific imports.